### ISSUE TITLE
helm 24.1.1 doesn't work with RDS 16.2

### ISSUE THREAD

**Issue by mkyc** [reportportal/reportportal#2288]
**Description:**
I tried to run report portal helm chart version 24.1.1 with AWS RDS 16.2 and it stops on migrations job. Migration job log says that it was able to connect

```shell
wait-for-it.sh: my-name.lalala.region.rds.amazonaws.com:5432 is available after 0 seconds   
```

but then it stops and no `migrations` are performed and show in log and it looks like it time outs. Then when `api` component (or any other) start, hikari connects, tomcat initialized, and usual stuff, and then (with first query I guess): 

```shell
....
ERROR: relation "public.integration_type" does not exist 
....
```

I connected to database and it's empty, so basically migrations failed. 

**Versions:**
RP 5.11.1
Helm 24.1.1

**Fun part:**
With RDS 13.15 it works. I didn't test what is the newest version it works with.

**Comment by Pink-Bumblebee** [comment]
@mkyc , your experience has just confirmed that ReportPortal is not fully compatible with RDS v.16. We are currently working on this problem, so it is need to wait till official release.
Anyway thank you for your information regarding compatibility with RDS 13.15. I hope that it will helps to other RP users.

**Comment by hlebkanonik** [comment]
Hello @mkyc, please share the complete migration pod logs. Also, check what is in the schema_migraions database. It should have version = 81 and dirty = false.

**Comment by mkyc** [comment]
@Pink-Bumblebee @hlebkanonik 

from ❗correctly performed❗  migration with RDS `13.15`:

```shell
wait-for-it.sh: waiting for opensearch-cluster-master.reportportal.svc.cluster.local:9200 without a timeout
wait-for-it.sh: waiting for my-name.lalala.region.rds.amazonaws.com:5432 without a timeout
wait-for-it.sh: my-name.lalala.region.rds.amazonaws.com:5432 is available after 0 seconds
0/u extensions (150.259813ms)
1/u initialize_schema (328.623695ms)
2/u initialize_quartz_schema (371.80785ms)
3/u default_data (395.83139ms)
4/u size_limitations (415.153125ms)
5/u test_case_id_type (445.312775ms)
6/u retries_handling (496.209053ms)
7/u auth_integration (535.974975ms)
8/u sender_case_enabled_field (615.069495ms)
9/u analyzer_params (628.14622ms)
10/u attachment_size (671.052052ms)
11/u password_encoding (582.944216ms)
12/u remove_ticket_duplicates (418.706499ms)
13/u add_allocated_storage_per_project (390.079814ms)
14/u test_case_id_size_increase (380.464652ms)
15/u statistics_decreasing (418.316166ms)
16/u remove_unused_indexes (402.19921ms)
17/u status_enum_extension (371.390945ms)
18/u job_attributes (315.372287ms)
19/u retries_handling_extension (299.956209ms)
20/u deep_merge_statistics_handling (301.477704ms)
21/u deep_merge_retries_fix (301.814197ms)
22/u deep_merge_nested_steps_fix (253.014749ms)
23/u rerun_item_statistics_fix (302.340342ms)
24/u widget_views_cleanup (306.698814ms)
25/u deep_merge_nested_steps_path_fix (306.816015ms)
26/u retries_lock_fix (303.540434ms)
27/u add_project_id_log (302.060159ms)
28/u create_log_project_idx (294.89826ms)
29/u create_table_item_project (294.900441ms)
30/u item_project_fill_part1 (291.040923ms)
31/u item_project_fill_part2 (290.889172ms)
32/u create_item_idx (290.119995ms)
33/u drop_log_message_trgm_idx (289.150545ms)
34/u fill_project_id_part1 (238.946344ms)
35/u fill_project_id_part2 (275.391443ms)
36/u drop_item_project (274.271242ms)
37/u create_log_message_trgm_idx (233.31712ms)
38/u add_not_null_log_project_id (233.620162ms)
39/u attachment_creation_date (235.034247ms)
40/u attachment_creation_date_fill (262.51073ms)
41/u attachment_creation_date_not_null (215.288386ms)
42/u shedlock_table (218.230794ms)
43/u attachment_for_deletion_table (283.999016ms)
44/u remove_triggers (285.281919ms)
45/u add_jobs_indexes (286.561991ms)
46/u job_values_bound (286.702243ms)
47/u quartz_remove_triggers (291.187515ms)
48/u composite_attribute (372.507576ms)
49/u analyzer_attribute (373.654687ms)
50/u bug_trend_content_fields (372.895449ms)
51/u cluster (351.656796ms)
52/u analyzer_search_attribute (391.863552ms)
53/u onboarding (390.65507ms)
54/u analyzer_unique_error_attribute (324.257042ms)
55/u onboarding_fix (370.84715ms)
56/u retries (371.773249ms)
57/u fix_deadlock (331.830295ms)
58/u alter_ticket (327.893948ms)
59/u stale_materialized_view (248.747167ms)
60/u sender_case_operator (287.53302ms)
61/u remove_acl (291.218445ms)
62/u remove_dashboard_cascade_drop (285.939155ms)
65/u launch_attribute_rules_length (294.508577ms)
66/u onboarding_update (291.608309ms)
67/u api_keys (295.89102ms)
68/u api_key_last_used_at (246.982311ms)
69/u replace_activity_table (250.567025ms)
70/u trigger_on_user_delete (274.291073ms)
71/u user_bid_inviting_user_id (278.643375ms)
72/u add_attachment_name (285.46016ms)
73/u sender_case_rule_name (246.92638ms)
74/u issue_type_hex_color (242.320216ms)
75/u issue_sub_type_hex_color (263.221217ms)
76/u user_bid_extension (216.419198ms)
77/u email_server_documentation_link (216.039214ms)
78/u drop_redundant_index (212.820132ms)
79/u drop_oauth_access_token (255.22632ms)
80/u update_onboarding (248.165742ms)
81/u remove_trgm_index (238.731291ms)
wait-for-it.sh: opensearch-cluster-master.reportportal.svc.cluster.local:9200 is available after 42 seconds
Template 'logs' created.
```

in ❗ incorrect run❗ with RDS `16.2` it was like this: 

```shell
wait-for-it.sh: waiting for opensearch-cluster-master.reportportal.svc.cluster.local:9200 without a timeout
wait-for-it.sh: waiting for my-name.lalala.region.rds.amazonaws.com:5432 without a timeout
wait-for-it.sh: my-name.lalala.region.rds.amazonaws.com:5432 is available after 0 seconds
error: pq: no pg_hba.conf entry for host "x.x.x.x", user "postgres", database "reportportal", no encryption
wait-for-it.sh: opensearch-cluster-master.reportportal.svc.cluster.local:9200 is available after 202 seconds
Template 'logs' already exists.
```

those two are exactly the same helm charts `24.1.1` while first one (correct one) is connected to RDS 13.15, and second (incorrect) is RDS 16.2. 

@hlebkanonik  regarding `public.schema_migrations` content - database was completely empty with `16.2` run, so nothing to share from that incorrect runs. In correct run, I can see `81, false` as you'd assumed.

**Comment by hlebkanonik** [comment]
Seems like a problem here:
```
error: pq: no pg_hba.conf entry for host "x.x.x.x", user "postgres", database "report portal", no encryption
```

Review the RDS parameter group and attempt to change `rds.force_ssl` to `0`. I found a similar issue related to the encryption algorithm at https://github.com/dbeaver/dbeaver/issues/21616#issuecomment-1793466865. It might be helpful.

**Comment by Pink-Bumblebee** [comment]
@mkyc , have you resolved your issue?

**Comment by mkyc** [comment]
@Pink-Bumblebee sorry I didn't have time. I guess @hlebkanonik guess might be correct. I don't think I'm going to be able to perform test in next couple weeks, so if you're unhappy with dangling issue, please close it. 

Anyways that should be tested by someone and added to documentation because error message doesn't say anything helpful right now.

**Comment by Pink-Bumblebee** [comment]
Ok, so I'm closing this issue.

**Comment by l0c0luke** [comment]
I can confirm that `rds.force_ssl = 0` is the fix for this and i am on rds postgres 16.3
